src/frontend/src/icp/services/ck.services.ts

```diff
@@ -69,12 +69,17 @@ export const convertCkErc20ToErc20 = async ({
 
 	// 1. Approve fees on ckETH Ledger for minter
 
+	assertNonNullish(
```

Comment:
NTH: for readability reasons I would insert the assertion before above comment `// 1. Approve ...`

Reply:
done